misc/version_updater.py

```diff
@@ -414,7 +414,12 @@ def refresh_npm_package_lock(update_files: set[Path]) -> set[Path]:
     },
     ROOT_DIR / "Cargo.toml": {
         Tool.License: [TOML_LICENSE_FIELD],
-        Tool.Parsec: [ReplaceRegex(r'^version = ".*"$', 'version = "{version}"')],
+        Tool.Parsec: [
+            ReplaceRegex(
+                r'^version = ".*" (\s*)# __PARSEC_VERSION__$',
+                'version = "{version}"' + " " * 35 + "# __PARSEC_VERSION__",
```

Comment:
35 blanks ??? I don't find this more readable than with a single blank
```toml
version = "3.0.0-rc.1+dev"                                   # __PARSEC_VERSION__
version = "3.0.0-rc.1+dev" # __PARSEC_VERSION__
```

and it makes the code here more complex:
```python
'version = "{version}"' + " " * 35 + "# __PARSEC_VERSION__",
```
vs
```python
'version = "{version}" # __PARSEC_VERSION__',
```

Reply:
It's `taplo` that align the comment to column 62 because of the authors line that end at col 60

Reply:
```toml
[workspace.package]
authors = [
    "@Scille/parsec-dev",       # Every parsec dev
    "@Scille/rust-code-owners", # And the code owners
]
edition = "2021"
homepage = "https://parsec.cloud"
license = "BUSL-1.1"
publish = false
repository = "https://github.com/Scille/parsec-cloud"
version = "3.0.0-rc.1+dev" # __PARSEC_VERSION__
```

I can propose this format, we keep 1 space between the version and comment but we need to have the authors entries on multiple line

Reply:
Adding a comment before `version` field solve this issue